Resources/Private/Scripts/ColorPickerEditor/src/NeosColorPicker/Alpha.js

```diff
@@ -0,0 +1,113 @@
+import React, { Component, PureComponent } from 'react'
+import reactCSS from 'reactcss'
+import * as alpha from 'react-color/lib/helpers/alpha'
+
+import Checkboard from 'react-color/lib/components/common/Checkboard'
+
+export class Alpha extends (PureComponent || Component) {
```

Comment:
Would be good to define the possible proptypes here

Reply:
Fixed